src/test/java/io/jenkins/plugins/checks/api/ChecksActionTest.java

```diff
@@ -13,5 +13,8 @@ void shouldConstructCorrectly() {
         assertThat(action.getLabel()).isPresent().hasValue("re-run");
         assertThat(action.getDescription()).isPresent().hasValue("re-run the Jenkins build");
         assertThat(action.getIdentifier()).isPresent().hasValue("re-run id");
+
+        assertThat(action.toString()).isEqualTo("ChecksAction{"
```

Comment:
I generally prefer to not test `toString` since it is a debugging output that might change frequently. However, since you already wrote the tests it is ok to let them here. In this case you also do not need to add the `@Generated` annotation.

```suggestion
        assertThat(action).hasToString("ChecksAction{"
```